component.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/86189ed851.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="main.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <!-- ---navbar---- -->
+
+    <nav class="navbar" >
+      <a href="index.html" class="Logo" href="#"><strong>YASHODA</strong></a>
+      <ul class="navbar-list">
+        <li class="navbar-list-item"><a href="index.html">Home</a></li>
+        <li class="navbar-list-item icons">
+          <a href="./component.html">Documentation</a>
+        </li>
+        <li class="navbar-list-item">
+          <a href="https://github.com/yashodaAgarwal">
+            <i class="fab fa-github icons"></i>
+          </a>
+        </li>
+
+        <li class="navbar-list-item">
+          <a href="https://www.linkedin.com/in/yashoda-agarwal-74994b18b">
+            <i class="fab fa-linkedin icons"></i>
+          </a>
+        </li>
+        <li class="navbar-list-item">
+          <a href="https://twitter.com/agarwal_yashoda?s=11">
+            <i class="fab fa-twitter icons"></i>
+          </a>
+        </li>
+      </ul>
+    </nav>
+
+    <!-- ---- sidebar ---- -->
+    <div class="sidebar">
+      <h3>Components</h3>
+      <ul class="nav">
+        <li>
+          <a href="#avatars">
+            <span>Avatar</span>
+          </a>
+        </li>
+        <li>
+          <a href="#alerts">
+            <span>Alert</span>
+          </a>
+        </li>
+        <li>
+          <a href="#badges">
+            <span>Badge</span>
+          </a>
+        </li>
+        <li>
+          <a href="buttons">
+            <span>Button</span>
+          </a>
+        </li>
+        <li>
+          <a href="#cards">
+            <span>Card</span>
+          </a>
+        </li>
+        <li>
+          <a href="#images">
+            <span>Image</span>
+          </a>
+        </li>
+        <li>
+          <a href="#Inputs">
+            <span>Input</span>
+          </a>
+        </li>
+      </ul>
+    </div>
+
+    <!-- ----- avatar ----- -->
+
+    <section class="main-title" id="avatars">
+      <h3 class="avatar-heading">Avatar</h3>
+      <p class="component-navbar-item">
+        In computing, an avatar (also known as a profile picture or userpic) is
+        a graphical representation of a user or the user's character or persona.
+      </p>
+      <div class="avatar-items">
+        <img src="./Asset/kitty.png" alt="" class="kitty avatar-xl" />
+        <img src="./Asset/kitty.png" alt="" class="kitty avatar-l" />
+        <img src="./Asset/kitty.png" alt="" class="kitty avatar-md" />
+        <img src="./Asset/kitty.png" alt="" class="kitty avatar-s" />
```

Comment:
'l' and 's' in the class name doesn't convey its use instead use 'avatar-lg' and  'avatar-sm'

Reply:
Okay sure I will change that.Thankyou

Reply:
I have made the necessary changes. Can you please check on that